examples/2_advanced/plot_s4_galaxies.py

```diff
@@ -0,0 +1,166 @@
+'''
+Stage IV Space Satellite Galaxy Survey
+======================================
+
+This example simulates a galaxy catalogue from a Stage IV Space Satellite Galaxy Survey such as
+*Euclid* and *Roman* combining the :ref:`sphx_glr_examples_1_basic_plot_density.py` and
+:ref:`sphx_glr_examples_1_basic_plot_lensing.py` examples with generators for
+the intrinsic galaxy ellipticity and the resulting shear with some auxiliary functions.
+
+The focus in this example is mock catalogue generation using auxiliary functions
+built for simulating Stage-IV galaxies.
+'''
+
+# %%
+# Setup
+# -----
+# The basic setup of galaxies and weak lensing fields is the same as in the
+# previous examples.
+#
+# In addition to a generator for intrinsic galaxy ellipticities,
+# following a normal distribution, we also show how to use auxiliary functions
+# to generate photometric redshift distributions and visibility masks.
```

Comment:
Maybe better to go with the generic "tomographic" since nothing here is specifically photometric.

```suggestion
# to generate tomographic redshift distributions and visibility masks.
```